packages/mantine/src/components/header/Header.tsx

```diff
@@ -42,11 +42,11 @@ export interface HeaderProps extends StylesApiProps<HeaderFactory>, Omit<GroupPr
      */
     borderBottom?: boolean;
     /**
-     * Use the modal variant when displaying the header inside a modal
+     * Use the secondary variant when displaying the header not at the top of the page
```

Comment:
I think the description is a bit hard to understand
```suggestion
     * Use the primary variant for page header and secondary variant elsewhere
```